src/Enums.ts

```diff
@@ -171,7 +171,7 @@ const toTrimmedLowerCase = (a: string | number) =>
         Option.some(a),
         Option.refine(String.isString),
         Option.map(flow(String.trim, String.toLowerCase)),
-        Option.defaultValue(a)
+        Option.defaultValue<string | number>(a)
```

Comment:
type inference improvements actually caught this bug 😅

Reply:
hrm.. well actually I guess what it was doing was inferring the `string | number` type from `a` instead of using the result of the prior `Option.map`. Either way, this seems clearer.